app/upgrades/v7/upgrades.go

```diff
@@ -37,7 +37,10 @@ func CreateUpgradeHandler(
 		// here when migrating (is it is not customized).
 		params := keepers.WasmKeeper.GetParams(ctx)
 		params.CodeUploadAccess = wasmtypes.AllowNobody
-		keepers.WasmKeeper.SetParams(ctx, params)
+		err = keepers.WasmKeeper.SetParams(ctx, params)
+		if err != nil {
+			return newVM, err
+		}
```

Comment:
lets just comment this out

Reply:
reverted and just silenced the lint check https://github.com/osmosis-labs/osmosis/pull/6413/commits/e04d32f9e7353ac863dc78243e86abc77729f06d